packages/core/src/domain/session/sessionStore.ts

```diff
@@ -20,6 +21,7 @@ export interface SessionStore {
   expireObservable: Observable<void>
   expire: () => void
   stop: () => void
+  updateSessionInStore: (state: Partial<SessionState>) => void
```

Comment:
When you use this method (e.g in `sessionManager.ts`), you already have the store context quite explicit: `sessionStore.updateSessionInStore` hence I think `sessionStore.updateSession()` is clear enough, don't you think?

Reply:
Yes, it makes sense. But when using it in the `recorderApi`, we don't have that context. Now is should be good since I renamed the exposed method from `rumSessionManager` to `setTrackingType`.

Suggestion applied ✔️